assignment3.txt

```diff
@@ -0,0 +1,10 @@
+Ava N.
+June 30, 2016
+
+Assignment 3
+
+1. The code written was invalid because the printf statement should look like this: printf("%d\n",i);
+2. i=(int) should be before the for loop. Also, the statement should look like this: int i;
```

Comment:
Not quite! Check again and see if you can find the two things wrong with this code snippet! If you still can't find them, let me know!